pkg/kubernetes/kubernetes.go

```diff
@@ -1033,8 +1033,23 @@ func (k *Kubernetes) DeleteEverest(ctx context.Context, namespace string, versio
 
 // GetDBNamespaces returns a list of namespaces that are monitored by the Everest operator.
 func (k *Kubernetes) GetDBNamespaces(ctx context.Context) ([]string, error) {
-	// List all namespaces managed by everest.
+	namespaces := []corev1.Namespace{}
+	// List Everest DB namespaces.
 	namespaceList, err := k.ListNamespaces(ctx, metav1.ListOptions{
+		LabelSelector: metav1.FormatLabelSelector(&metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				common.EverestDBNamespaceLabel: "",
+			},
+		}),
+	})
+	if err != nil {
+		return nil, errors.Join(err, errors.New("failed to get watched namespaces"))
+	}
+	namespaces = append(namespaces, namespaceList.Items...)
+
+	// Prior to 1.3.0, we used the app.kubernetes.io/managed-by label to identify namespaces managed by Everest.
+	// We still support the use of this label for backwards compatibility.
+	namespaceList, err = k.ListNamespaces(ctx, metav1.ListOptions{
```

Comment:
The reason for making this change is that the `app.kubernetes.io/managed-by` label is also used by Helm. As a result, after installing DB namespaces via Helm, they go missing in the UI because this label is taken over by Helm..